src/mono/mono/metadata/bundled-resources.c

```diff
@@ -31,8 +34,10 @@ mono_bundled_resources_free (void)
 {
 	g_assert (mono_runtime_is_shutting_down ());
 
-	g_hash_table_destroy (bundled_resources);
+	dn_simdhash_free (bundled_resources);
+	dn_simdhash_free (bundled_resource_key_lookup_table);
```

Comment:
you dynamically allocate the keys for `bundled_resource_key_lookup_table` but don't `g_free` them

Reply:
But also we don't have any callers. So I guess this all just leaks anyway